app/src/main/res/layout/fraction_interaction_item.xml

```diff
@@ -32,14 +31,22 @@
         android:digits="0123456789/- "
         android:enabled="@{!viewModel.isReadOnly}"
         android:focusable="true"
+        android:fontFamily="sans-serif"
         android:hint="@string/fractions_hint_text"
-        android:inputType="text"
-        android:longClickable="false"
         android:imeOptions="actionDone"
+        android:inputType="date"
```

Comment:
One problem with this date input type. It allows user to add `-` and if this is submitted it will crash.

Reply:
Fraction can be negative. If its is getting crashed need to find the issue in parsing now updated by ben.

Reply:
Okay just make a note of it, that we need to resolve this. Beside this looks good to me.